src/benchmark/index.ts

```diff
@@ -35,12 +36,19 @@ const newtonsMethodFunc = {
     func: [newtonsMethodWhile, newtonsMethod],
 };
 
+const quicksortFunc = {
+    arguments: [new Array(1000000), 0, 99],
```

Comment:
The second and third argument is not necessary, because the `quickSortWhile` function only uses the first one.